messages/messages.go

```diff
@@ -72,7 +72,7 @@ var (
 	SETTINGS_PATH = "Error looking for settings.yaml"
 
 	NA                          = "N/A"
-	VALID_DOCKERFILE_BASE_IMAGE = "astronomerinc/ap-airflow"
+	VALID_DOCKERFILE_BASE_IMAGE = "quay.io/astronomer/ap-airflow"
```

Comment:
Could this be moved into `houston/constants.go` or a new file either`config/constants.go` or `airflow/constants.go` (We're trying to come up with a better way to organize the constants right now.).

Can you also use pascal case (we're converting all of the constants to match to follow golang guidelines: `VaildDockerfileBaseImage`).

Then we can use the const value instead of the hard coded string value in the other implementation and test locations.

@andriisoldatenko what do you think about the constant value location?

Reply:
i think PR LGTM for now, we can refactor it later.

Reply:
My changes here are only related to the location of the docker images. If we want to make changes to the structure of the code or the naming of the variables, that should be done in a different branch.